### PR TITLE
rootless: drop dependency on docker

### PR DESCRIPTION
**Pull request by giuseppe**
use directly the definition in "syscall".

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

File: pkg/rootless/rootless_linux.go [hunk 1]
```diff
@@ -501,7 +500,7 @@ func becomeRootInUserNS(pausePid, fileToRead string, fileOutput *os.File) (bool,
 	defer gosignal.Reset()
 	go func() {
 		for s := range c {
-			if s == signal.SIGCHLD || s == signal.SIGPIPE {
```
Comment (mheon) [inline pkg/rootless/rootless_linux.go]: Should we use x/sys/unix instead of syscall? I think unix is more maintained

Comment (baude) [reply pkg/rootless/rootless_linux.go]: might as well given we are editing, nice change @giuseppe

**Comment by rhatdan** [comment]
I like the title on this PR. But lets switch to unix.

File: pkg/rootless/rootless_linux.go (hunk 1 above)
Comment (giuseppe) [reply pkg/rootless/rootless_linux.go]: sure.  I've updated the patch to replace syscall with x/sys/unix

**Comment by rhatdan** [comment]
/hold
/lgtm
hold cancel when tests pass.

**Comment by TomSweeneyRedHat** [comment]
Happy green test buttons now.

**Pull request merged**